acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java

```diff
@@ -139,4 +146,30 @@ public static boolean isMinus(String minus) {
         }
     }
 
+    public static RPCHook getAclRPCHook(String fileName) {
+        JSONObject yamlDataObject = null;
+        try {
+            yamlDataObject = AclUtils.getYamlDataObject(fileName,
+                JSONObject.class);
+        } catch (Exception e) {
+            e.printStackTrace();
+            return null;
```

Comment:
use logger instead of print

Reply:
Okay,I will adjust my codes in this pr again.